test/e2e/providers/azure/azure.go

```diff
@@ -18,7 +18,7 @@ const (
 	defaultCredentialsSecretName = "azure-cloud-credentials"
 	defaultImageOffer            = "WindowsServer"
 	defaultImagePublisher        = "MicrosoftWindowsServer"
-	defaultImageSKU              = "datacenter-core-20h2-with-containers-smalldisk"
+	defaultImageSKU              = "2019-Datacenter-with-Containers"
```

Comment:
Isn't this [going away](https://techcommunity.microsoft.com/t5/containers/important-update-deprecation-of-docker-virtual-machine-images/ba-p/3646272) later this year? We might as well switch to a non-Docker image now instead of having to make a change again in April. This should be called out in the commit message.